docs/tools/Deploy-IRIS-Monitor.md

```diff
@@ -1,62 +1,56 @@
-# How to deploy IRIShub Monitor
+# How to deploy IRIS Monitor
 
 Please make sure that iris is installed in your computer and added to $PATH.You can see this page for insturcion https://github.com/irisnet/irishub. You also need /bin/bash, wc, ps to ensure the monitor work properly.
 
-### Get Monitor Info
-
-* Address of validator
+## Start IRIS Monitor
 
 ```
--a= < hex-encoded-address-of-validator >
+irismon --address=EAC535EC37EB3AE8D18C623BA4B4C8128BC082D2 --account-address=faa1nwpzlrs35nawthal6vz2rjr4k8xjvn7k8l63st --chain-id=irishub-stage --node=http://localhost:36657
 ```
 
-* Chain-ID
-```
---chain-id= < id-of-blockchain >
-```
+Parameters：
 
-* Node
+- `address`：hex encoded validator address
+- `account-address`：bech32 encoded account address
+- `chain-id`：blockchain id that you want to monitor
+- `node`：listening address of the node that you want to monitor ("tcp://localhost:26657" by default, you should not change this if you didn't modify your rpc port)
```

Comment:
The rpc url of a irishub node, default value is `tcp://localhost:26657`. If you want to monitor other irishub nodes, please change this value.